policyengine_us/parameters/gov/states/oh/tax/income/deductions/unreimbursed_medical_care_expenses/rate.yaml

```diff
@@ -0,0 +1,15 @@
+description: Ohio reduces the Unreimbursed Medical Care Expenses by the following percentage of the federal AGI.
+values:
+  2021-01-01: 0.075
```

Comment:
i believe in the legal code this refers to the federal rate - there is a parameter for it under irs, deductions, itemized, medical